src/pages/MonthlyGood/MonthlyGoodSetupPage.vue

```diff
@@ -808,8 +837,22 @@ export default {
 		}
 	}
 
+	.auto-lending-notice {
```

Comment:
Hacky space adjustment. The messaging does look cramped without the additional margins